internal/services/grpc_gateway/middleware/cors/cors_test.go

```diff
@@ -5,10 +5,16 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"testing"
-
-	http2 "github.com/kwilteam/kwil-db/internal/services/grpc_gateway/middleware/http-dummy"
```

Comment:
I figure I'll follow up with a PR to remove all grpc code and protos.  Since 0.9 introduced new RPC functionality on the JSON-RPC service that would be a waste of time to implement on the grpc server (and grpc http gateway), this means we're definitely removing it for 0.9